app/javascript/tests/subject/CaseStatus.test.js

```diff
@@ -0,0 +1,225 @@
+        expect(wrapper.find(Form.Control).exists()).toBeTruthy();
+        expect(wrapper.find('option').length).toEqual(6);
+        caseStatusValues.forEach(function(value, index) {
+            expect(wrapper.find('option').at(index).text()).toEqual(value);
+        });
+    });
+
+    it('Changing Case Status opens modal', () => {
+        const wrapper = getWrapper(mockPatient1, false);
+        expect(wrapper.find(Modal).exists()).toBeFalsy();
+        wrapper.find(Form.Control).simulate('change', { target: { id: 'case_status', value: 'Confirmed' }, persist: jest.fn() });
+        expect(wrapper.find(Modal).exists()).toBeTruthy();
+    });
+
+    it('Properly renders modal', () => {
+        const wrapper = getWrapper(mockPatient1, false);
+        wrapper.find(Form.Control).simulate('change', { target: { id: 'case_status', value: 'Confirmed' }, persist: jest.fn() });
+        expect(wrapper.find(Modal.Title).exists()).toBeTruthy();
+        expect(wrapper.find(Modal.Title).text()).toEqual('Case Status');
+        expect(wrapper.find(Modal.Body).exists()).toBeTruthy();
+        expect(wrapper.find(Modal.Footer).exists()).toBeTruthy();
+        expect(wrapper.find(Button).at(0).text()).toEqual('Cancel');
+        expect(wrapper.find(Button).at(1).text()).toEqual('Submit');
+    });
+
+    it('Properly renders modal body and updates state when not monitoring', () => {
+        const wrapper = getWrapper(mockPatient3, false);
+        wrapper.find(Form.Control).simulate('change', { target: { id: 'case_status', value: 'Confirmed' }, persist: jest.fn() });
+        const modalBody = wrapper.find(Modal.Body);
+
+        expect(wrapper.state('showCaseStatusModal')).toBeTruthy();
+        expect(wrapper.state('case_status')).toEqual('Confirmed');
+        expect(wrapper.state('confirmedOrProbable')).toBeTruthy();
+        expect(modalBody.find('p').text()).toEqual(`Are you sure you want to change case status from ${mockPatient3.case_status} to Confirmed? Since this record is on the Closed line list, updating this value will not move this record to another line list. If this individual should be actively monitored, please update the record’s Monitoring Status.`);
+    });
+
+    it('Properly renders modal body and updates state when changing Case Status to blank', () => {
+        const wrapper = getWrapper(mockPatient1, false);
+        wrapper.find(Form.Control).simulate('change', { target: { id: 'case_status', value: '' }, persist: jest.fn() });
+        const modalBody = wrapper.find(Modal.Body);
+
+        expect(wrapper.state('showCaseStatusModal')).toBeTruthy();
+        expect(wrapper.state('case_status')).toEqual('');
+        expect(wrapper.state('confirmedOrProbable')).toBeFalsy();
+        expect(modalBody.find('p').text()).toEqual(`Are you sure you want to change case status from ${mockPatient1.case_status} to blank? The monitoree will remain in the same workflow.`);
+    });
```

Comment:
Verify that the line list and workflow of the record has not been changed in this test as well.

Reply:
done